update introduction content and URLs in README

graphitefriction opened the pull request.

graphitefriction commented:
@mojavelinux I forgot about the whole "also homepage" thing, so please make sure I didn't bork that up. I did edit the intro content some to bring it in line with the new content in the Asciidoctor docs index and features pages. I know we also want to pull out the installation details and link to the docs instead, but that is definitely for a different PR :sweat_smile:

Mogztter commented on README.adoc:
```diff
@@ -17,133 +17,124 @@ ifdef::env-github[]
-The Asciidoctor project is {uri-repo}[hosted on GitHub].
+{url-project}[Asciidoctor] is a fast, open source, Ruby-based text processor for parsing {url-what-is-asciidoc}[AsciiDoc] into a document model and converting it to output formats such as HTML 5, DocBook 5, manual pages, PDF, EPUB 3, and other formats.
+Asciidoctor also has an ecosystem of extensions, converters, build plugins, and tools to help you author and publish AsciiDoc.
+You can find the documentation for these projects elsewhere on this site or in the {url-org)[Asciidoctor organization on GitHub].
```
```suggestion
You can find the documentation for these projects elsewhere on this site or in the {url-org}[Asciidoctor organization on GitHub].
```

mojavelinux commented on README.adoc:
```diff
@@ -17,133 +17,124 @@ ifdef::env-github[]
-Asciidoctor is written in Ruby and runs on all major operating systems.
-The Asciidoctor project is {uri-repo}[hosted on GitHub].
+{url-project}[Asciidoctor] is a fast, open source, Ruby-based text processor for parsing {url-what-is-asciidoc}[AsciiDoc] into a document model and converting it to output formats such as HTML 5, DocBook 5, manual pages, PDF, EPUB 3, and other formats.
+Asciidoctor also has an ecosystem of extensions, converters, build plugins, and tools to help you author and publish AsciiDoc.
```
```suggestion
Asciidoctor also has an ecosystem of extensions, converters, build plugins, and tools to help you author and publish content written in AsciiDoc.
```

mojavelinux commented on README.adoc:
```diff
@@ -462,18 +457,18 @@ Let's work together to make this a welcoming, professional, inclusive, and safe
 == Trademarks
 
-AsciiDoc is a Trademark of the Eclipse Foundation, Inc.
+AsciiDoc is a trademark of the Eclipse Foundation.
```
Is dropping the ", Inc" part of the transition of the Eclipse Foundation to Europe? If so, we need to make the same change to the docs UI.

graphitefriction replied:
I got clarification from legal. Initially, I made this change because in the Eclipse handbook and many of the other projects' notices, they format/phrase the attribution without the "Inc". But you were correct, it should be "Eclipse Foundation, Inc." Also, AsciiDoc is a registered trademark, so attribution requires the (R) as well as the first/most prominent usage of it, which I added in the first paragraph.  AsciiDoc Language (when writing about as the specification project, which we don't do on this page but I added attribution anyway) gets the (TM).

graphitefriction replied:
BTW, you can find the full list of trademarks here: https://www.eclipse.org/legal/trademarks.php#attribution and the usage guidelines for them here: https://www.eclipse.org/legal/logo_guidelines.php

mojavelinux commented on README.adoc:
```diff
@@ -462,18 +457,18 @@ Let's work together to make this a welcoming, professional, inclusive, and safe
+*Asciidoctor* is led by https://github.com/mojavelinux[Dan Allen] and https://github.com/graphitefriction[Sarah White] and has received contributions from {url-contributors}[many individuals] in Asciidoctor's awesome community.
+The project was initiated in 2012 by https://github.com/erebor[Ryan Waldron] and based on a prototype written by https://github.com/nickh[Nick Hengeveld].
 
 *AsciiDoc* was started and maintained by Stuart Rackham from https://github.com/asciidoc/asciidoc/blob/master/CHANGELOG.txt[2002 to 2013] and has received contributions from many individuals in the https://github.com/asciidoc/asciidoc/graphs/contributors[AsciiDoc community].
```
```suggestion
*AsciiDoc.py* was started and maintained by Stuart Rackham from https://github.com/asciidoc-py/asciidoc-py2/blob/master/CHANGELOG.txt[2002 to 2013] and has received contributions from many individuals in the https://github.com/asciidoc-py/asciidoc-py2/graphs/contributors[AsciiDoc.py community].
```

mojavelinux commented on README.adoc:
```diff
@@ -462,18 +457,18 @@ Let's work together to make this a welcoming, professional, inclusive, and safe
-*Asciidoctor* is led by https://github.com/mojavelinux[Dan Allen] and https://github.com/graphitefriction[Sarah White] and has received contributions from {uri-contributors}[many individuals] in Asciidoctor's awesome community.
-The project was initiated in 2012 by https://github.com/erebor[Ryan Waldron] and based on {uri-prototype}[a prototype] written by https://github.com/nickh[Nick Hengeveld].
+*Asciidoctor* is led by https://github.com/mojavelinux[Dan Allen] and https://github.com/graphitefriction[Sarah White] and has received contributions from {url-contributors}[many individuals] in Asciidoctor's awesome community.
+The project was initiated in 2012 by https://github.com/erebor[Ryan Waldron] and based on a prototype written by https://github.com/nickh[Nick Hengeveld].
```
```suggestion
The project was initiated in 2012 by https://github.com/erebor[Ryan Waldron] based on a prototype written by https://github.com/nickh[Nick Hengeveld] for the Git website.
```

mojavelinux commented on README.adoc:
```diff
@@ -349,15 +337,18 @@ For instance, to write the file to a different directory, use:
 === Ruby API
 
+Asciidoctor also provides an API.
+The API is intended for integration with other Ruby software, such as Rails, Sinatra and GitHub, and other languages, such as Java (via AsciidoctorJ) and JavaScript (via Asciidoctor.js).
```
```suggestion
The API is intended for integration with other Ruby software, such as Rails, GitHub, and GitLab, as well as other languages, such as Java (via AsciidoctorJ) and JavaScript (via Asciidoctor.js).
```

mojavelinux commented on README.adoc:
```diff
@@ -159,11 +150,11 @@ Asciidoctor works best when you use UTF-8 everywhere.
-Asciidoctor can be installed using (a) package managers for popular Linux distributions, (b) Homebrew for macOS, (c) the `gem install` command (recommended for Windows users or if you'll be installing additional gems), (d) the Asciidoctor Docker image, or (e) Bundler.
+Asciidoctor is packaged and distributed to RubyGems.org as a {url-rubygem}[RubyGem named asciidoctor^].
+The asciidoctor gem can be installed on all major operating systems using Ruby packaging tools (gem or bundle).
+Asciidoctor is also distributed as a Docker image and as a package for numerous Linux distributions as well as macOS.
```
```suggestion
Asciidoctor is also distributed as a Docker image, as a package for numerous Linux distributions, and as a package for macOS (via Homebrew).
```

mojavelinux commented on README.adoc:
```diff
@@ -17,133 +17,124 @@ ifdef::env-github[]
-{uri-opal}[Opal] is used to transcompile the Ruby source to JavaScript to produce {uri-asciidoctorjs}[Asciidoctor.js].
-Asciidoctor.js is a fully-functional version of Asciidoctor that works in any JavaScript environment, such as a web browser or Node.js.
-It's used to power the AsciiDoc preview extensions for Chrome, Atom, Brackets and other web-based tooling.
+Asciidoctor is the successor to AsciiDoc.py (aka AsciiDoc Python).
```
```suggestion
Asciidoctor is the successor to AsciiDoc.py.
```

mojavelinux commented on README.adoc:
```diff
@@ -17,133 +17,124 @@ ifdef::env-github[]
 
-Asciidoctor reads and parses text written in the AsciiDoc syntax, then feeds the parse tree to a set of built-in converters to produce HTML5, DocBook 5, and man(ual) page output.
-You have the option of using your own converter or loading {uri-tilt}[Tilt]-supported templates to customize the generated output or produce additional formats.
+Asciidoctor provides {url-docs}/asciidoctor/latest/converters/[converters] for three output formats by default: {url-docs}/asciidoctor/latest/html-backend/[HTML 5], {url-docs}/asciidoctor/latest/docbook-backend/[DocBook 5], and {url-docs}/asciidoctor/latest/manpage-backend/[manpage] (manual page).
```
```suggestion
Asciidoctor provides {url-docs}/asciidoctor/latest/converters/[converters] for three output formats by default: {url-docs}/asciidoctor/latest/html-backend/[HTML 5], {url-docs}/asciidoctor/latest/docbook-backend/[DocBook 5], and {url-docs}/asciidoctor/latest/manpage-backend/[manpage] (manual page).
Additional converters are provided by separate gems.
```

mojavelinux commented on README.adoc:
```diff
@@ -17,133 +17,124 @@ ifdef::env-github[]
-The Asciidoctor project is {uri-repo}[hosted on GitHub].
+{url-project}[Asciidoctor] is a fast, open source, Ruby-based text processor for parsing AsciiDoc(R) into a document model and converting it to output formats such as HTML 5, DocBook 5, manual pages, PDF, EPUB 3, and other formats.
+Asciidoctor also has an ecosystem of extensions, converters, build plugins, and tools to help you author and publish content written in {url-what-is-asciidoc}[AsciiDoc].
+You can find the documentation for these projects elsewhere on this site or in the {url-org}[Asciidoctor organization on GitHub].
```
```suggestion
You can find the documentation for these projects at {url-docs}.
```

mojavelinux commented on README.adoc:
```diff
@@ -17,133 +17,124 @@ ifdef::env-github[]
-Asciidoctor is written in Ruby and runs on all major operating systems.
-The Asciidoctor project is {uri-repo}[hosted on GitHub].
+{url-project}[Asciidoctor] is a fast, open source, Ruby-based text processor for parsing AsciiDoc(R) into a document model and converting it to output formats such as HTML 5, DocBook 5, manual pages, PDF, EPUB 3, and other formats.
+Asciidoctor also has an ecosystem of extensions, converters, build plugins, and tools to help you author and publish content written in {url-what-is-asciidoc}[AsciiDoc].
```
Let's put an empty line after the first sentence to make the preamble shorter. It displays quite prominently on asciidoctor.org and is running too long as a result.

mojavelinux commented on README.adoc:
```diff
@@ -17,133 +17,124 @@ ifdef::env-github[]
 
-In addition to the classic AsciiDoc syntax, Asciidoctor recognizes additional markup and formatting options, such as font-based icons (e.g., `+icon:fire[]+`) and UI elements (e.g., `+button:[Save]+`).
-Asciidoctor also offers a modern, responsive theme based on {uri-foundation}[Foundation] to style the HTML5 output.
+Although Asciidoctor is written in Ruby, it does not mean you have to use Ruby.
```
```suggestion
Although Asciidoctor is written in Ruby, it does not mean you need Ruby to use it.
```

mojavelinux commented on README.adoc:
```diff
@@ -462,18 +457,18 @@ Let's work together to make this a welcoming, professional, inclusive, and safe
 Copyright (C) 2012-2021 Dan Allen, Sarah White, Ryan Waldron, and the individual contributors to Asciidoctor.
```
```suggestion
Copyright (C) 2012-present Dan Allen, Sarah White, Ryan Waldron, and the individual contributors to Asciidoctor.
```

mojavelinux commented:
@graphitefriction I was able to verify that these changes to not break the asciidoctor.org homepage (aside from the point I made about the lead paragraph in the preamble).

I may have gone overboard with my changes. If you think anything needs to be deferred, I'm happy to move the suggestions to a separate PR and sync them with the docs. Just let me know.

graphitefriction commented:
> I was able to verify that these changes to not break the asciidoctor.org homepage (aside from the point I made about the lead paragraph in the preamble).

Excellent! And thank you! I added the line after the first sentence in the preamble.

> I may have gone overboard with my changes. If you think anything needs to be deferred, I'm happy to move the suggestions to a separate PR and sync them with the docs. Just let me know.

They all looked good to me, so I added them in (and squished them together :sandwich: ). I think this file is ready to go now.

Mogztter commented:
Well done @graphitefriction 👌🏻 
I will update the 🇫🇷 version with your edits!
